app/assets/stylesheets/application.css.scss

```diff
@@ -135,3 +163,46 @@ input#front-search-button {
   text-align: center;
   color: white !important;
 }
+
+.tag {
+  color: blue;
+}
+
+.venues {
+  width: 60%;
+  background: {
+    color: #170563;
+  }
+  margin: 8% auto;
+  opacity: .94;
+  text-align: center;
+}
+
+.bands {
+  width: 60%;
+  background: {
+    color: #170563;
+  }
+  margin: 8% auto;
+  opacity: .94;
+  text-align: center;
+}
+
+.review-form {
+  float: left;
+  width: 60%;
+}
+
+.reviews {
+  float: left;
+  width: 60%;
+  margin-left: 1%;
+  margin-top: 3%;
+  background: {
+    color: #170563;
```

Comment:
Color literals like `#170563` should only be used in variable declarations; they should be referred to via variable everywhere else.